package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "Backend server for analytics",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
this is supposed to change when we merge the tests in right? Because currently it's "test": "jest" on branch 23 to make the test work.

Reply:
Yes exactly, when you finish your testing scripts you can add it to the scripts JSON